### DISCUSSION TITLE
Daily Task Indicator addition: Tears of Guthix

### DISCUSSION THREAD

**Discussion by MsN0VA** [runelite/runelite#15931] [Ideas]
Would be nice to add Tears of Guthix on an infobox like the other dailies. Right now, it only shows on the chat as a notification but no "icon" shows on screen to remind you while you are playing (let's be real, we don't read the game chat that much).

**Comment by Enriath** [comment]
The Daily Tasks Indicator plugin does not do infobox reminders, just chat messages. In that regard, the game itself has reminders that you can enable by talking to Juna.
